dbg.h

```diff
@@ -453,6 +457,16 @@ inline bool pretty_print(std::ostream& stream, const std::tuple<>&) {
   return true;
 }
 
+template <>
+inline bool pretty_print(std::ostream& stream, const time&) {
+  const auto ms =  std::chrono::duration_cast<std::chrono::microseconds>(std::chrono::high_resolution_clock::now().time_since_epoch()).count() % 1000000;
+  const auto hms = std::chrono::system_clock::to_time_t(std::chrono::system_clock::now());
+  const std::tm* tm = std::localtime(&hms);
+  stream << std::put_time(tm, "%H:%M:%S") << '.'
+         << ms ;
```

Comment:
This will print a wrong value if `ms` has a value smaller than 100000, because the leading zero(s) are missing. If `ms = 12345`, it should print `HH:MM:SS.012345`